### PR TITLE
Fix a typo in aten_xla_type.cpp

### PR DESCRIPTION
**Pull request by bhavya01**

### CODE REVIEW

**Review by JackCaoG — APPROVED** [review]

**Review by wonjoolee95 — APPROVED** [review]

**Comment by bhavya01** [comment]
Can I get write access to pytorch/xla repo? I can't merge the PR without that.

**Comment by JackCaoG** [comment]
granted.

**Pull request merged**
